### PR TITLE
Loki: remove live option for logs panel

### PR DESCRIPTION
**Pull request by davkal**
The live option for Loki in the logs panels caused a couple of issues:

- Log Panel: Live tailing recreates websocket on refresh #19532
- Log Panel: Selecting Live doesn't trigger a preview. #19530

This PR is a quick-fix to prevent live panel related issues from impacting too many people.

Speaking to the Loki team, they don't see a use for a live option in the dashboards to begin with, since it will be out of sync with the rest of the dashboard.

### CODE REVIEW

**Comment by aocenas** [comment]
@davkal We should probably also change the log panel docs too as they mention the live option. Otherwise looks good.

**Comment by aocenas** [comment]
Hmm also what will happen if user sets something live and then updates, probably no way to turn it back off, only recreating the whole panel?

**Comment by davkal** [comment]
> Hmm also what will happen if user sets something live and then updates, probably no way to turn it back off, only recreating the whole panel?

Another good point, I'm happy to live with this for now.

**Review by aocenas — APPROVED** [review]

**Comment by wuestkamp** [comment]
The live feature was awesome, any plan integrating this again? :)

**Comment by RamiBerm** [comment]
I'm forced to stay at 6.4.1 because of this

**Comment by davkal** [comment]
The live feature is still available in Explore. It was a bit unstable in logs panels so we'd like to learn a bit more about the problems. 
For those who have been using this for a while, what were the biggest issues? @wuestkamp @RamiBerm

**Comment by wuestkamp** [comment]
Ah ok I see. I only used it through Grafana Explore and it always worked fine, never used the live feature in a Panel. I thought this ticket would also remove this from Explore.

**Comment by RamiBerm** [comment]
@davkal Biggest issues for me are the panel seemingly losing connection at times and not automatically scrolling to new logs when display order is in descending mode

**Comment by cwilliams5** [comment]
The Grafana docs still say this is possible. Pulled my hair out looking for the option. Just submitted PR #22704 to update docs.

Not sure if this is the place to discuss use cases, but we would definitely like this back. It's part of why I set up Loki in the first place. To the comment that it will be out of sync with the rest of the dashboard, that's expected. But if I have a realtime metrics board refreshing every 1s, with a grafana log panel by some other metrics,  I can watch the CPU use of loki spike to meet the demand. It instead I do a live tail in explore, the cpu use of loki is practically 0 and i get my live tail.

TLDR its still useful for realtime dashes and hope it can be brought back.
